test/fixtures/mandate-fixtures.js

```diff
@@ -0,0 +1,120 @@
+'use strict'
+
+const Payer = require('../../common/classes/Payer.class')
+const Mandate = require('../../common/classes/Mandate.class')
+const GatewayAccount = require('../../common/classes/GatewayAccount.class')
+const Service = require('../../common/classes/Service.class')
+
+const randomExternalId = () => Math.random().toString(36).substring(7)
+const randomNumber = () => Math.round(Math.random() * 10000) + 1
+const randomUrl = () => 'https://' + randomExternalId() + '.com'
+
+module.exports = {
+  validTokenExchangeResponse: (opts = {}) => {
+    const data = {
+      external_id: opts.external_id || randomExternalId(),
+      mandate_reference: opts.mandate_reference || 'buy Silvia a coffee',
+      type: opts.type || 'CHARGE',
+      state: opts.state || randomNumber(),
+      return_url: opts.return_url || randomUrl(),
+      gateway_account_external_id: opts.gateway_account_external_id || randomExternalId(),
+      gateway_account_id: opts.gateway_account_id || randomNumber()
+    }
+    return {
+      getPlain: () => {
+        return data
+      }
+    }
+  },
+  validGatewayAccountResponse: (opts = {}) => {
+    const data = {
+      gatewayAccountId: opts.gatewayAccountId || randomNumber(),
+      gatewayAccountExternalId: opts.gatewayAccountExternalId || randomExternalId(),
+      paymentMethod: opts.paymentMethod || 'DIRECT_DEBIT',
+      paymentProvider: opts.paymentProvider || 'SANDBOX',
```

Comment:
is there any reason why these fields are camel case but the ones in validTokenExchangeResponse are snake case? if this is meant to represent https://github.com/alphagov/pay-direct-debit-connector/blob/master/src/main/java/uk/gov/pay/directdebit/gatewayaccounts/api/GatewayAccountResponse.java should it be snake case?

Reply:
Oops, this was meant to be a file rename but somehow i lost the deletion of payments-fixtures.js somewhere. So I haven't changed this. But you're right looks like it should be snake case